### PR TITLE
Fix tests for setting registery

### PR DESCRIPTION
**Pull request by tbrisker**

### CODE REVIEW

**Comment by lzap** [comment]
Discovery is broken AF, before:

    220 runs, 449 assertions, 1 failures, 167 errors, 0 skips

after this patch

    220 runs, 449 assertions, 6 failures, 57 errors, 0 skips

I guess that's a pass.

File: test/test_helper_discovery.rb [hunk 1]
```diff
@@ -52,22 +52,22 @@ def extract_form_errors(response)
 end
 
 def set_default_settings
```
Comment (ezr-ondrej) [inline test/test_helper_discovery.rb:54]: This should not be needed now :tihinking:

File: test/test_helper_discovery.rb [hunk 2]
```diff
@@ -52,22 +52,22 @@ def extract_form_errors(response)
...
-  FactoryBot.create(:setting, :name => 'discovery_always_rebuild_dns', :value => true, :category => 'Setting::Discovered')
-  FactoryBot.create(:setting, :name => 'discovery_error_on_existing', :value => false, :category => 'Setting::Discovered')
-  FactoryBot.create(:setting, :name => 'discovery_naming', :value => 'Fact', :category => 'Setting::Discovered')
-  FactoryBot.create(:setting, :name => 'discovery_auto_bond', :value => false, :category => 'Setting::Discovered')
+  Setting['discovery_fact'] = 'discovery_bootif'
+  Setting['discovery_hostname'] = 'discovery_bootif'
+  Setting['discovery_auto'] = true
+  Setting['discovery_reboot'] = true
+  Setting['discovery_organization'] = "Organization"
+  Setting['discovery_location'] = "Location"
```
Comment (lzap) [inline test/test_helper_discovery.rb:60]: Damn this should be "Organization 1". Will do a followup PR.

Comment (lzap) [reply test/test_helper_discovery.rb:60]: Fixed in https://github.com/theforeman/foreman_discovery/commit/8bbab181d2e2623b9d1a70782bbcfb78e8fca87e

**Pull request merged**